### ISSUE TITLE
NTR, pulmo: Elevated level of eosinophilic granulocytes

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5563]
**Preferred term label:**
Elevated level of eosinophilic granulocytes

**Synonyms**
Elevated level of eosinophils in broncho-alveolar lavage fluid

**Definition (free text, please give PubMed ID)**
Usually, eosinophils make up less than 0.5% of all cells found in the broncho-alveloar lavage fluid. But in eosinophilic lung disease, the eosinophil cell proportion typically represents more than 25%. Comment: An elevated level of eosinophil cells are also a result of infections, or an allergic reaction or can be drug-induced. 
(Costabel ISBN0412792702; PMID: 3396403; PMID: 10678650)

**Parent term (use hpo.jax.org/app)**
Abnormal cellular composition of bronchoalveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 178500, OMIM:  607685 , ORPHA:183, Idiopathic Acute Eosinophilic Pneumonia (PubMed: 12403693)

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by GrieseAG** [comment]
Preferred term label:
Elevated level of eosinophils granulocytes

**Comment by pnrobinson** [comment]
Label -- Increased eosinophil proportion in bronchoalveolar lavage

**Comment by pnrobinson** [comment]
@mgriese , @katrinknoflach
There is an existing term
https://hpo.jax.org/app/browse/term/HP:0032071

Should we make the existing term refer just to pulmonary infiltration and remove the reference to BAL there? And then create this term?

**Comment by GrieseAG** [comment]
Yes in the existing term, the reference to BAL should be removed. Then we could have two seperate terms 1) pulmonary eosinophilic infiltration of the airways > detected by BAL (and biopsy) = elevated level of eosionophilic granulocytes in BAL and 2) tissue eosinophilic infiltration (Synonym tissue eosinophilia) > can only be detected by biopsy.

**Comment by pnrobinson** [comment]
Pulmonary eosinophila
-- tissue
-- lavage
